curriculum/js1/6.js

```diff
@@ -5,8 +5,15 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, possibleDivisor = a -1, sum = 0) => {
+  if (possibleDivisor === 1) {
+    return sum;
+  }
+  if (a % possibleDivisor === 0) {
+    sum = sum + possibleDivisor;
+  }
+  possibleDivisor = possibleDivisor - 1;
+  return solution(a, possibleDivisor, sum);
```

Comment:
This works. I can only offer options for shortening the code. If you want, you could do this arithmetic inside of the function call:
```javascript
return solution(a, possibleDivisor - 1, sum);
```